server/lib/amplitude.js

```diff
@@ -133,7 +135,7 @@ const transform = initialize(SERVICES, EVENTS, FUZZY_EVENTS);
 module.exports = receiveEvent;
 
 function receiveEvent (event, request, data) {
-  if (! event || ! request || ! data) {
+  if (! amplitude.disabled || ! event || ! request || ! data) {
```

Comment:
We should remove the `!` here since we would want it to return if the `amplitude.disabled=true`

Reply:
This is probably the cause of some of your test failures.